Add section about Template naming and locations to PAGEVIEW (#1546)
TYPO3-Documentation/TYPO3CMS-Reference-Typoscript: Documentation/ContentObjects/Pageview/Index.rst

    :language: typoscript
    :caption: EXT:my_sitepackage/Configuration/TypoScript/setup.typoscript

To ensure proper rendering, the following requirements and
fallback mechanisms apply:

**Target Path Structure**:

> *   The target directory **must contain a folder named `Pages/`**.

Reviewer: ```suggestion
*   The target directory **must contain a folder named :directory:`Pages/`**.
```

> *   This folder must contain Fluid templates in *.html format.

Reviewer: ```suggestion
*   This folder must contain Fluid templates in :file`*.html` format.
```

**Template Naming Convention**:

> *   The Fluid template file should match the name of the selected backend layout

Reviewer: Should we drop a word about the misleading term "backend layout" here and mention it's really more a "page layout"? :-)

Reviewer 2: We have these messages all over and everyone is still confused. It should be changed in the core.

    of your current page.
*   The first letter of the Fluid template file must be **uppercase**.

**Fallback Behavior**:

> *   If page record does not specify a backend layout, TYPO3 tries to detect

Reviewer: ```suggestion
*   If the page record does not specify a backend layout, TYPO3 tries to detect
```

>     next-level backend layouts by travelling up the page rootline.

Reviewer: ```suggestion
    next-level backend layouts by traversing up the page rootline.
```

*   If still no backend layout could be found, TYPO3 falls back to `Default.html`.

> *   If you explizit set backend layout to `none` a Fluid template

Reviewer: ```suggestion
*   If you explicitly set the backend layout to `none`, a Fluid template
```

>     with name `None.html` will be used.

Reviewer: ```suggestion
    with name :file:`None.html` will be used.
```

    :language: typoscript
    :caption: EXT:my_sitepackage/Configuration/TypoScript/setup.typoscript

..  _cobj-pageview-template-naming:

Template naming and locations
=============================

To ensure proper rendering, the following requirements and
fallback mechanisms apply:

..  _cobj-pageview-template-naming-path:

Target path structure
---------------------

*   The target directory **must contain a folder named :directory:`Pages/`**.
*   This folder must contain Fluid templates in :file`*.html` format.

> ..  _cobj-pageview-template-naming-cobnventions:

Author: ```suggestion
..  _cobj-pageview-template-naming-conventions:
```

Template naming convention
--------------------------

*   The Fluid template file **must** match the name of the selected backend layout
    of your current page.
*   The first letter of the Fluid template file must be **uppercase**.

..  _cobj-pageview-template-naming-fallback:

> Fallback behavior when not backend layout is defined

Author: ```suggestion
Fallback behavior when no backend layout is defined
```
